Eastern-Research-Group/mywaterway
# Feature/re organize cypress tests

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3331144

## Main Changes:
* Reorganizes  Cypress tests from directories to flat files for each route tested.
* Adds in missing "Aquatic Life" tests to the top level topic pages.
* Installs the latest version of Cypress (and supporting packages) to the top level "orchestrating" application, and adds necessary configuration.
* Moves e2e tests from "client" app to top level orchestrating app.
* Removes no longer needed Cypress (and supporting packages) and config from the "client" app.

## Steps To Test:
1. Remove no longer used packages from the **client** app:    
from `app/client` directory run: `npm ci`
2. Install new packages in **orchestrating app**:    
from `app` directory run: `npm ci`
3. Run Cypress e2e tests and ensure tests still work as before:    
from `app` directory run: `npm run cypress`
